compose/src/main/java/com/bottlerocketstudios/compose/filterdropdown/FilterDropDownMenu.kt

```diff
@@ -0,0 +1,102 @@
+package com.bottlerocketstudios.compose.filterdropdown
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.Card
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.unit.toSize
+import com.bottlerocketstudios.compose.resources.brown_grey
+import com.bottlerocketstudios.compose.util.Preview
+
+@Composable
+fun FilterDropDownMenu(
```

Comment:
@Br-DiorD @BottleRocket-Colin How much should we rely on theme/style for widgets like this? I see a lot of areas where sizing, padding, color, and style values could be used from `Dimens`, `Colors`, and `MaterialTheme` here to better support additional form factors/dark mode/etc (similar to `AuthCodeScreen`)?

Thoughts?